django_pgschemas/management/commands/migrateschema.py

```diff
@@ -26,6 +26,8 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         runschema = NonInteractiveRunSchemaCommand()
         options.pop("run_syncdb", False)
+        if "skip_checks" not in options:
+            options["skip_checks"] = True
```

Comment:
Shouldn't this default to false?

Reply:
[The definition](https://github.com/django/django/blob/77d455ae73b177a32102f0b248828b5d63c0aa24/django/core/management/base.py#L366) of that argument has `store_true` so it defaults to False. I think that in the implementation you copied as being part of the `call_comand` method makes sense to default to True because it will probably run in a context where the checks are not needed or already checked